dts/bindings/iio/adc/nuvoton,npcx-adc.yaml

```diff
@@ -22,6 +22,10 @@ properties:
         type: int
         required: true
         description: the offset of threshold detector register address
+    threshold-count:
+        type: int
+        required: true
+        description: the number of adc threshold detector
```

Comment:
Please reword this; I don't understand what it means

Reply:
Done.